src/ff1/alloc.rs

```diff
@@ -21,6 +20,24 @@ fn pow(x: u32, e: usize) -> BigUint {
     res
 }
 
+/// Extension trait adding FF1-relevant methods to `BigUint`.
+trait Numeral {
+    /// Type used for byte representations.
+    type Bytes: AsRef<[u8]>;
+
+    /// Returns the integer interpreted from the given bytes in big-endian order.
+    fn from_bytes(s: impl Iterator<Item = u8>) -> Self;
+
+    /// Returns the big-endian byte representation of this integer.
+    fn to_bytes(&self, b: usize) -> Self::Bytes;
+
+    /// Compute (self + other) mod radix^m
```

Comment:
```suggestion
    /// Compute `(self + other) mod radix^m`, where other is represented
    /// as a big-endian byte sequence.
```

Reply:
This comment change is incorrect; `other` here has type `Self`.

Reply:
```suggestion
    /// Computes `(self + other) mod radix^m`.
```